speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/DecisionTuple.cpp

```diff
@@ -2,12 +2,46 @@
 
 DecisionTuple::DecisionTuple()
```

Comment:
This file could use some more documentation explaining what a DecisionTuple is

Reply:
this is done in refactor